system_baseline/openapi/api.spec.yaml

```diff
@@ -297,6 +302,35 @@ components:
               value:
                 type: string
   parameters:
+    orderByParam:
+      name: order_by
+      in: query
+      required: false
+      schema:
+        default: "display_name"
+        type: string
+        enum:
+          - display_name
+          - created_on
+      description: Ordering field name, defaults to display_name
+    orderHowParam:
+      name: order_how
+      in: query
+      required: false
+      schema:
+        default: "ASC"
+        type: string
+        enum:
+          - ASC
+          - DESC
+      description: Direction of the ordering, defaults to ASC
+    displayNameParam:
+      name: display_name
```

Comment:
Is the idea to start simple with specific search fields vs. having a generic search field that can accept different search types? e.g. `search='display_name = foo'` `'search='display_name ~ foo'`

Does the API specification for platform talk about search?

Reply:
its currently exact only, but I'm guessing a library exists already to do more complex searches. I'll take a look tomorrow